test/entity.spec.js

```diff
@@ -346,6 +346,64 @@ describe('JHipster generator for entity', () => {
                     assert.jsonFileContent('.jhipster/Foo.json', { clientRootFolder: 'test-root' });
                 });
             });
+
+            describe('with authority as relationship', () => {
+                before(done => {
+                    helpers
+                        .run(require.resolve('../generators/entity'))
+                        .inTmpDir(dir => {
+                            fse.copySync(path.join(__dirname, '../test/templates/default-ng2'), dir);
```

Comment:
You set authority relationship as oauth2 only.
 You should move to a oauth2 template or enabling authority relationship with session.